docs/running-jobs/interactive-jobs.md

```diff
@@ -31,9 +31,11 @@ To run an interactive GUI application on HPC Systems, we must install an X windo
 #### Windows setup
 
 On Windows we must first install an X windows server application to allow the cluster to forward the GUI information to your local system. For Windows, we will use an application called Xming to accomplish
-this. [Download the Xming here](https://sourceforge.net/projects/xming/).
+this. [Download Xming here](https://sourceforge.net/projects/xming/).
 
-Next we must enable X11 forwarding on the PuTTY application. Download and install the [PuTTY application](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) here if you have not done so already.
+##### PuTTY
```

Comment:
This is confusing. Is PuTTY required for this or not? I think the confusion will be cleared by including PuTTY in your overview paragraph.